client/components/vote-escrow/LockupsTable.tsx

```diff
@@ -66,7 +66,7 @@ const LockupsTable: FunctionComponent = () => {
           <tr>
             <th>OGV</th>
             <th>Time remaining</th>
-            <th>Lockup ends</th>
+            <th>End of lock-up</th>
```

Comment:
We would also need the conditional statement here to ensure we render the right heading for the row.

Reply:
I think if you can make 100% sure the table has only 1 row, conditional statement should be implemented. If the lockups table might have more than 2 rows, this will not make sense. 
But I guess the lockup contract looks similar to curve voting escrow which allows only 1 lock per address. If this is our case, I will add conditional statement as well.

Reply:
Ah, yes, you're right, our contracts allow multiple lock ups so we need the heading to make sense no matter the value in the row(s) beneath.

Suggestion: 

Instead of "None" we could use "-" which, alongside the Unstake button becoming active, will make it obvious which stakes have no duration remaining at a glance.

Reply:
Nice! do I have to keep `End of lock-up` or revert it `Lockup ends` ?

Reply:
I would revert 🤝

Reply:
Resolved in 944cfdebe344e4d2a04f4f62f6bb550764f562b0